airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2982,6 +2982,13 @@ components:
           format: uri
         icon:
           type: string
+        normalizationRepository:
+          type: string
+        normalizationTag:
+          type: string
+        supportDbt:
+          type: boolean
+          default: false
```

Comment:
I think this api is called when adding custom destinations in OSS... do we need to add these to the form in the frontend?

What happens if a destination doesn't have these fields set? Are we properly handling this as a destination that doesn't run normalization?